File: LeetCode/1684.Count_Consistent_Strings/main.py
#!/usr/bin/python
from solution import Solution

# Unit Test

solutionObject = Solution()
allowed = "ab"
words = ["ad","bd","aaab","baa","badab"]
if (s.countConsistentStrings(allowed, words) == 2):
    print("PASS")
else:
    print("FAILED")

File: LeetCode/1684.Count_Consistent_Strings/main.cpp
#include "solution.h"

// Unit Test

int main(){
    Solution s;
    string allowed = "ab";
    vector<string> words {"ad","bd","aaab","baa","badab"};
    if (s.countConsistentStrings(allowed, words) == 2) { cout << "PASS" << endl;}
    else {cout<<"FAILED"<<endl;}
    return 0;
}